frontend/src/test/scala/bloop/bsp/BspCompileSpec.scala

```diff
@@ -576,17 +570,10 @@ class BspCompileSpec(
         assertSameExternalClassesDirs(thirdCompiledState, compiledState, projects)
         assertNoDiff(
           thirdCompiledState.lastDiagnostics(`A`),
-          """#3: task start 3
-            |  -> Msg: Start no-op compilation for a
-            |  -> Data kind: compile-task
-            |#3: a/src/A.scala
+          """#3: a/src/A.scala
             |  -> List()
             |  -> reset = true
-            |#3: task finish 3
-            |  -> errors 0, warnings 0
-            |  -> Msg: Compiled 'a'
-            |  -> Data kind: compile-report
-            """.stripMargin
+            """.stripMargin // no-op so it only gives the compile-report
```

Comment:
Actually, compile report is in the task finish, so we only get the compile response, no? But that should be fine as nothing really changes

Reply:
Ahh, yea sorry that's my mistake. I was switching up the names of the compile response and compile report. You're correct, it's only the compile _response_ that is now returned.